File: CPP/lab/student.cpp
#include<iostream>
using namespace std;
class Student{
    private:
        int id, marks;
        string name;
    public:
        Student(){
            id = 0;
            marks = 0;
            name = "";
        }
        void accept()
        {
            cout<<"Enter Student ID: ";
            cin>>id;
            cout<<"Enter Student Marks:";
            cin>>marks;
            cin.ignore();
            cout<<"Enter Student Name:";
            getline(cin, name);
        }
        void display()
        {
            cout<<"ID: "<<id<<endl;
            cout<<"Marks: "<<marks<<endl;
            cout<<"Name: "<<name<<endl;
            cout<<"\n======================\n";
        }
};
int main()
{
    Student s[3];
    for(int i=0;i<3;i++)
    {
        cout<<"Enter Student Details: "<<i+1<<endl;
        s[i].accept();
    }
    for(int i=0;i<3;i++)
    {
        s[i].display();
    }
    return 0;
}

File: Core-Java/JAVA_ASSIGNMENTS/src/com/app/stud/Student.java
package com.app.stud;

public class Student {

    private int studId;
    private String name;
    private double m1;
    private double m2;
    private double m3;

    public Student(int studId, String name, double m1, double m2, double m3) {
        this.studId = studId;
        this.name = name;
        this.m1 = m1;
        this.m2 = m2;
        this.m3 = m3;
    }

    public void display() {
        System.out.println("Student Details");
        System.out.println("--------------------");
        System.out.println("Student Id : " + studId);
        System.out.println("Name       : " + name);
        System.out.println("M1         : " + m1);
        System.out.println("M2         : " + m2);
        System.out.println("M3         : " + m3);
    }
}